rust/package/agama.changes

```diff
@@ -1,3 +1,16 @@
+-------------------------------------------------------------------
+Thu Feb 29 09:49:18 UTC 2024 - Ladislav Slezák <lslezak@suse.com>
+
+- Web server:
+  - Accept also IPv6 connections
```

Comment:
This was done in #1057

Reply:
so please mention it in change with `gh#openSUSE/agama#1057` annotation.